Contrib/ReleaseHighlight.md

```diff
@@ -1,31 +1,33 @@
 ## Release Highlights
-🔒 Set minimum coinjoin input count
-🔒 Stricter absolute limits for maximum coinjoin mining fee rate and minimum coinjoin input count
-🔒 Prevent solo coinjoining
-👀 Coordinator Connection String
-👀 Advanced send Workflow
+
+🥕 Taproot receive addresses
+🚀 Faster transaction broadcasting
+⏫ More accurate CPFP fee estimation
+📉 Safer protocol
+🪲 Backend optimizations
+
 ## Release Summary
 
-Wasabi Wallet v2.1.0.0 adds safeguards for coinjoin participants.
-Clients can now set their own policies regarding minimum input count and maximum coordination fee to protect from malicious coordinators which tries to charge more than they announce.
+Wasabi Wallet v2.2.0.0 release
+
+This version of Wasabi is full of extra features, improvements, and fixes. Here’s what got updated in today’s release:
 
-### Apply coinjoin coordination policies to all rounds:
+🥕 Taproot receive addresses
 
-It was possible for a malicious coordinator to bypass the client policies for blame round and set different coordination fees for those.
-v2.1.0.0 fixes this issue by enforcing the policies to blame rounds too.
+Users can now choose taproot when generating a new receiving address. Spending a taproot input is 16% cheaper compared to spending a native segwit input. Users aren’t the only ones that benefit - Bitcoin nodes can verify taproot’s signatures faster than legacy ones, so the entire network gets a little speed boost.
 
-### Stricter safeguards:
+🚀 Faster transaction broadcasting
 
-Wasabi clients use more restrictive policies that guarantee that even in presence of a malicious coordinator, the risk of losing money is mitigated. These are: absolute maximum coordination fee rate changed from 0.01 to 0.005, absolute minimum input count changed from 2 to 5.
+Spending your coins is not only cheaper in this release, it’s also faster! Transactions are now broadcast to multiple nodes in parallel (through the Tor network) and fail more quickly if the transaction is invalid. Users who upgrade will notice it now takes half as long to send their coins.
```

Comment:
```suggestion
A more sophisticated transaction broadcasting mechanism was introduced to have faster transaction broadcasting. Transactions are now broadcast to multiple nodes in parallel (through the Tor network) and fail more quickly if the transaction is invalid. Users who upgrade will notice it now takes half as long to send their coins.
```

nice marketing phrase. seems a bit over exaggerated imo. As it's only the case if people use the taproot addr and even if they do they they will probably not notice the few sats savings.

`if the transaction is invalid`
this can be confusing what is meant here, maybe clarify

Reply:
I agree it was too enthusiastic for me as well, I had to think 5s about how it get cheaper then was disappointed :D